Source/compositorclient/RPI/ModeSet.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <stdint.h>
+
+class ModeSet
+{
+    public:
+        ModeSet(const ModeSet&) = delete;
+        ModeSet& operator= (const ModeSet&) = delete;
+
+        ModeSet();
+        ~ModeSet();
+
+    public:
+        const struct gbm_device* const UnderlyingHandle() const
```

Comment:
second const doesn't add any value and makes for some people the code harder to undertand I guess

Reply:
Someone has been on a C++ training :-) Will remove it.